app/routes/index/index.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  beforeModel() {
+  redirect() {
     this.transitionTo('index.mygoals');
   }
 });
```

Comment:
Why was this replaced from "/index.js" to "index/index.js"?
I understand the change to redirect instead of beforeModel(), but the replacement prevents the redirection